_events/2018-11-10-first_workshop.md

```diff
@@ -1,16 +1,138 @@
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6993.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6994.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_7002.jpg"
+  image5_path="/images/events/2018-11-10-first_workshop/DSC_7001.jpg"
+  image6_path="/images/events/2018-11-10-first_workshop/DSC_7006.jpg"
+  image7_path="/images/events/2018-11-10-first_workshop/DSC_7007.jpg"
+  image8_path="/images/events/2018-11-10-first_workshop/DSC_7009.jpg"
+%}
+
+At the end we quickly check the purity with a spectrophotometer and freezed the DNA until library preparation
+
+*The details of the protocol we used can be found in this website: [DNA extraction protocol]({% link _protocols/beer-dna-extraction.md %}).*
+
+## DNA sequencing
+
+During the DNA extracted, we started to play with the MinION with the installation of the Nanopore software and the test of flowcell. We then did the library preparation: preparation of the DNA for sequencing.
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6857.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6954.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6853.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6941.jpg"
+%}
+
+We then loaded the MinION with our DNA and launch the sequencing
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6951.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6946.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6973.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6981.jpg"
+%}
+
+It was quite exciting. We had long read until 24Kb, but the quality 
+started to decrease quite quickly (probably not enough DNA) so we stopped
+it earlier.
+
+*The details of the protocol we used can be found in this website: [DNA sequencing protocol]({% link _protocols/beer-dna-sequencing.md %}).*
+
+Milad then analyzed the generated sequences and extracted some *Saccharomyces cerevisae*.
+
+{% include one_central_image.html
+    image_path="/images/events/2018-11-10-first_workshop/analysis_taxonomy.png"
+%}
+
+Very promising results (given we stoped the sequencing)!
+
+## Next steps
+
+Our next challenge will be to simplify the DNA extraction protocol, specifically 
+using simpler and everyday life tools, and ensure its robustness on every location and 
+situation, i.e., the protocol has to work in a specialised lab as well as in a high 
+school chemistry/biology labs. To overcome this challenge, we will collect ideas from 
+Do-It-Yourself biology projects.
+
+We will make special efforts in making the protocols accessible and understandable 
+by everyone, with graphics supporting each step and explaining their importance and 
+their execution. 
+
+For the data analysis, a dedicated workflow will be available via [Galaxy](). 
```

Comment:
ping @mmiladi

Reply:
I would also link a history so we can share the data.

Reply:
Here is the link: https://usegalaxy.eu/u/milad/h/nanoporebeerdecodechimaytriple

Reply:
Merci!